docs/src/pages/component-demos/drawers/UndockedDrawer.js

```diff
@@ -26,28 +26,217 @@ const styleSheet = createStyleSheet('UndockedDrawer', () => ({
+        </Drawer>
+        <Drawer
+          anchor="top"
+          open={this.state.open.top}
+          onRequestClose={this.handleTopClose}
+          onClick={this.handleTopClose}
+        >
+          <List className={classes.listFull} padding={false}>
+            <ListItem button>
+              <ListItemIcon>
+                <InboxIcon />
+              </ListItemIcon>
+              <ListItemText primary="Inbox" />
+            </ListItem>
+            <ListItem button>
+              <ListItemIcon>
+                <StarIcon />
+              </ListItemIcon>
+              <ListItemText primary="Starred" />
+            </ListItem>
+            <ListItem button>
+              <ListItemIcon>
+                <SendIcon />
+              </ListItemIcon>
+              <ListItemText primary="Send mail" />
+            </ListItem>
+            <ListItem button>
+              <ListItemIcon>
+                <DraftsIcon />
+              </ListItemIcon>
+              <ListItemText primary="Drafts" />
+            </ListItem>
+          </List>
+          <Divider />
+          <List className={classes.listFull} padding={false}>
+            <ListItem button>
+              <ListItemIcon>
+                <MailIcon />
+              </ListItemIcon>
+              <ListItemText primary="All mail" />
+            </ListItem>
+            <ListItem button>
+              <ListItemIcon>
+                <DeleteIcon />
+              </ListItemIcon>
+              <ListItemText primary="Trash" />
+            </ListItem>
+            <ListItem button>
+              <ListItemIcon>
+                <ReportIcon />
+              </ListItemIcon>
+              <ListItemText primary="Spam" />
+            </ListItem>
+          </List>
+          <div className={classes.remainder} />
+        </Drawer>
+        <Drawer
+          anchor="bottom"
+          open={this.state.open.bottom}
+          onRequestClose={this.handleBottomClose}
+          onClick={this.handleBottomClose}
+        >
+          <List className={classes.listFull} padding={false}>
+            <ListItem button>
+              <ListItemIcon>
+                <InboxIcon />
+              </ListItemIcon>
+              <ListItemText primary="Inbox" />
+            </ListItem>
+            <ListItem button>
+              <ListItemIcon>
+                <StarIcon />
+              </ListItemIcon>
+              <ListItemText primary="Starred" />
+            </ListItem>
+            <ListItem button>
+              <ListItemIcon>
+                <SendIcon />
+              </ListItemIcon>
+              <ListItemText primary="Send mail" />
+            </ListItem>
+            <ListItem button>
+              <ListItemIcon>
+                <DraftsIcon />
+              </ListItemIcon>
+              <ListItemText primary="Drafts" />
+            </ListItem>
+          </List>
+          <Divider />
+          <List className={classes.listFull} padding={false}>
```

Comment:
I think that we should keep the children of the list DRY. Please use an intermediary constant.